src/robot/client.ts

```diff
@@ -39,7 +39,12 @@ interface WebRTCOptions {
   host: string;
   signalingAddress: string;
   rtcConfig: RTCConfiguration | undefined;
+  // RECONNECTION OPTIONS
   noReconnect?: boolean;
+  // value should be a positive int; default is 10
```

Comment:
These comments should show up in docs (you can check using `make build` and looking in the docs folder), which I think requires the `/** */` syntax. we also only export `DialWebRTCConf` so the comments would be more suited to show up there